node-exporter/main.libsonnet

```diff
@@ -0,0 +1,43 @@
+local k = import 'github.com/grafana/jsonnet-libs/ksonnet-util/kausal.libsonnet';
+
+{
+  new(image='prom/node-exporter:v1.1.2'):: {
+    local container = k.core.v1.container,
+    container::
+      container.new('node-exporter', image)
+      + container.withPorts(
+        k.core.v1.containerPort.new('http-metrics', 9100)
+      )
+      + container.withArgs([
+        '--path.procfs=/host/proc',
+        '--path.sysfs=/host/sys',
+
+        '--collector.netdev.device-exclude=^veth.+$',
+
+        // We run an older version due to the renamed metrics.  There ignores are from newer version.
+        '--collector.filesystem.ignored-fs-types=^(tmpfs|autofs|binfmt_misc|bpf|cgroup2?|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|mqueue|nsfs|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|selinuxfs|squashfs|sysfs|tracefs)$',
+        '--collector.filesystem.ignored-mount-points=^/(rootfs/)?(dev|proc|sys|var/lib/docker/.+)($|/)',
```

Comment:
@tomwilkie These have been around for ~3 years in prometheus-ksonnet and the comment does not make much sense considering we are running the latest. Do you know how relevant this still is?

Reply:
No you'll need to investigate that; they were added ~4 years ago in an effort to reduce cardinality.